inlong-sort/sort-connectors/mysql-cdc/src/main/java/org/apache/inlong/sort/cdc/mysql/source/split/MySqlSplitSerializer.java

```diff
@@ -104,6 +104,38 @@ public static Map<TableId, TableChange> readTableSchemas(int version, DataInputD
         return tableSchemas;
     }
 
+    public static void writeTableDdls(
+            Map<TableId, String> tableDdls, DataOutputSerializer out) throws IOException {
+        final int size = tableDdls.size();
+        out.writeInt(size);
+        for (Map.Entry<TableId, String> entry : tableDdls.entrySet()) {
+            out.writeUTF(entry.getKey().toString());
+            out.writeUTF(entry.getValue());
+        }
+    }
+
+    public static Map<TableId, String> readTableDdls(int version, DataInputDeserializer in)
+            throws IOException {
+        Map<TableId, String> tableDdls = new HashMap<>();
+        final int size = in.readInt();
+        for (int i = 0; i < size; i++) {
+            TableId tableId = TableId.parse(in.readUTF());
```

Comment:
You can ref to `readReadPhaseMetric` method to add `judge in.available() > 0`

Reply:
Thanks, it has been fixed.